Artifacts/windows-firefox/Artifactfile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Firefox",
+  "description": "Installs Firefox using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Firefox"
+  ],
+  "iconUri": "https://chocolatey.org/content/packageimages/Firefox.44.0.2.png",
+  "targetOsType": "Windows",
+   "parameters": {
+        "installUsername": {
```

Comment:
@mikaelkrief : I see that this artifact requires the user to pass in a username and password (for installing firefox under a specific user's credentials). Would it be possible to eliminate the need for these parameters?

The general pattern that most artifacts follow is to create a temporary admin account to install the chocolatey package (Example: https://github.com/Azure/azure-devtestlab/tree/master/Artifacts/windows-7zip). This way, the user is not required to pass in a user name and password. 

Will that work for this artifact?